09/dominik-cernelic--oh-cms-where-art-thou.md

```diff
@@ -0,0 +1,20 @@
+# Oh CMS, Where Art Thou?
```

Comment:
Can you remove this file? There is another one in the correct location (`meetups/2018/09`)